{
	"document_title": "Security update for file",
	"document_type": "SUSE Patch",
	"document_publisher": {
		"type": "Vendor",
		"contact_details": "security@suse.de",
		"issuing_authority": "SUSE Security Team"
	},
	"document_tracking": {
		"identification": {
			"id": "openSUSE-SU-2020:0677-1"
		},
		"status": "Final",
		"version": "1",
		"revision_history": {
			"revision": {
				"number": "1",
				"date": "2020-05-22T16:16:29Z",
				"description": "current"
			}
		},
		"initial_release_date": "2020-05-22T16:16:29Z",
		"current_release_date": "2020-05-22T16:16:29Z",
		"generator": {
			"engine": "cve-database/bin/generate-cvrf.pl",
			"date": "2017-02-24T01:00:00Z"
		}
	},
	"document_notes": [
		{
			"text": "Security update for file",
			"title": "Topic",
			"type": "Summary"
		},
		{
			"text": "This update for file fixes the following issues:\n\nSecurity issues fixed:\n\n- CVE-2019-18218: Fixed a heap-based buffer overflow in cdf_read_property_info() (bsc#1154661).\n\nNon-security issue fixed:\n\n- Fixed broken '--help' output (bsc#1169512).\n\nThis update was imported from the SUSE:SLE-15:Update update project.",
			"title": "Details",
			"type": "General"
		},
		{
			"text": "The CVRF data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
			"title": "Terms of Use",
			"type": "Legal Disclaimer"
		},
		{
			"text": "openSUSE-2020-677",
			"title": "Patchnames",
			"type": "Details"
		}
	],
	"documentdistribution": "Copyright SUSE LLC under the Creative Commons License 4.0 with Attribution (CC-BY-4.0)",
	"document_references": [
		{
			"type": "Self",
			"url": "https://lists.opensuse.org/opensuse-security-announce/2020-05/msg00044.html",
			"description": "E-Mail link for openSUSE-SU-2020:0677-1"
		},
		{
			"type": "Self",
			"url": "https://www.suse.com/support/security/rating/",
			"description": "SUSE Security Ratings"
		},
		{
			"type": "Self",
			"url": "https://bugzilla.suse.com/1154661",
			"description": "SUSE Bug 1154661"
		},
		{
			"type": "Self",
			"url": "https://bugzilla.suse.com/1169512",
			"description": "SUSE Bug 1169512"
		},
		{
			"type": "Self",
			"url": "https://www.suse.com/security/cve/CVE-2019-18218/",
			"description": "SUSE CVE CVE-2019-18218 page"
		}
	],
	"product_tree": {
		"branch": [
			{
				"type": "Product Family",
				"name": "openSUSE Leap 15.1",
				"branch": [
					{
						"type": "Product Name",
						"name": "openSUSE Leap 15.1",
						"full_product_name": {
							"text": "openSUSE Leap 15.1",
							"product_id": "openSUSE Leap 15.1",
							"cpe": "cpe:/o:opensuse:leap:15.1"
						}
					}
				],
				"full_product_name": {}
			},
			{
				"type": "Product Version",
				"name": "file-5.32-lp151.8.3.1",
				"full_product_name": {
					"text": "file-5.32-lp151.8.3.1",
					"product_id": "file-5.32-lp151.8.3.1"
				}
			},
			{
				"type": "Product Version",
				"name": "file-devel-5.32-lp151.8.3.1",
				"full_product_name": {
					"text": "file-devel-5.32-lp151.8.3.1",
					"product_id": "file-devel-5.32-lp151.8.3.1"
				}
			},
			{
				"type": "Product Version",
				"name": "file-devel-32bit-5.32-lp151.8.3.1",
				"full_product_name": {
					"text": "file-devel-32bit-5.32-lp151.8.3.1",
					"product_id": "file-devel-32bit-5.32-lp151.8.3.1"
				}
			},
			{
				"type": "Product Version",
				"name": "file-magic-5.32-lp151.8.3.1",
				"full_product_name": {
					"text": "file-magic-5.32-lp151.8.3.1",
					"product_id": "file-magic-5.32-lp151.8.3.1"
				}
			},
			{
				"type": "Product Version",
				"name": "libmagic1-5.32-lp151.8.3.1",
				"full_product_name": {
					"text": "libmagic1-5.32-lp151.8.3.1",
					"product_id": "libmagic1-5.32-lp151.8.3.1"
				}
			},
			{
				"type": "Product Version",
				"name": "libmagic1-32bit-5.32-lp151.8.3.1",
				"full_product_name": {
					"text": "libmagic1-32bit-5.32-lp151.8.3.1",
					"product_id": "libmagic1-32bit-5.32-lp151.8.3.1"
				}
			},
			{
				"type": "Product Version",
				"name": "python2-magic-5.32-lp151.8.3.1",
				"full_product_name": {
					"text": "python2-magic-5.32-lp151.8.3.1",
					"product_id": "python2-magic-5.32-lp151.8.3.1"
				}
			},
			{
				"type": "Product Version",
				"name": "python3-magic-5.32-lp151.8.3.1",
				"full_product_name": {
					"text": "python3-magic-5.32-lp151.8.3.1",
					"product_id": "python3-magic-5.32-lp151.8.3.1"
				}
			}
		],
		"relationship": [
			{
				"product_reference": "file-5.32-lp151.8.3.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Leap 15.1"
			},
			{
				"product_reference": "file-devel-5.32-lp151.8.3.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Leap 15.1"
			},
			{
				"product_reference": "file-devel-32bit-5.32-lp151.8.3.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Leap 15.1"
			},
			{
				"product_reference": "file-magic-5.32-lp151.8.3.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Leap 15.1"
			},
			{
				"product_reference": "libmagic1-5.32-lp151.8.3.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Leap 15.1"
			},
			{
				"product_reference": "libmagic1-32bit-5.32-lp151.8.3.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Leap 15.1"
			},
			{
				"product_reference": "python2-magic-5.32-lp151.8.3.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Leap 15.1"
			},
			{
				"product_reference": "python3-magic-5.32-lp151.8.3.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Leap 15.1"
			}
		]
	},
	"vulnerability": [
		{
			"notes": [
				{
					"text": "cdf_read_property_info in cdf.c in file through 5.37 does not restrict the number of CDF_VECTOR elements, which allows a heap-based buffer overflow (4-byte out-of-bounds write).",
					"title": "Vulnerability Description",
					"type": "General"
				}
			],
			"cve": "CVE-2019-18218",
			"product_statuses": [
				{
					"type": "Fixed",
					"product_id": [
						"openSUSE Leap 15.1:file-5.32-lp151.8.3.1",
						"openSUSE Leap 15.1:file-devel-32bit-5.32-lp151.8.3.1",
						"openSUSE Leap 15.1:file-devel-5.32-lp151.8.3.1",
						"openSUSE Leap 15.1:file-magic-5.32-lp151.8.3.1",
						"openSUSE Leap 15.1:libmagic1-32bit-5.32-lp151.8.3.1",
						"openSUSE Leap 15.1:libmagic1-5.32-lp151.8.3.1",
						"openSUSE Leap 15.1:python2-magic-5.32-lp151.8.3.1",
						"openSUSE Leap 15.1:python3-magic-5.32-lp151.8.3.1"
					]
				}
			],
			"threats": [
				{
					"type": "Impact",
					"description": "critical"
				}
			],
			"cvss_score_sets": {
				"score_set_v_2": {
					"base_score_v_2": "6.8",
					"vector_v_2": "AV:N/AC:M/Au:N/C:P/I:P/A:P"
				},
				"score_set_v_3": {
					"base_score_v_3": "9.8",
					"vector_v_3": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
				},
				"scoreset": {}
			},
			"remediations": [
				{
					"type": "Vendor Fix",
					"description": {
						"text": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
						"lang": "en"
					},
					"url": "https://lists.opensuse.org/opensuse-security-announce/2020-05/msg00044.html"
				}
			],
			"references": [
				{
					"url": "https://www.suse.com/security/cve/CVE-2019-18218.html",
					"description": "CVE-2019-18218"
				},
				{
					"url": "https://bugzilla.suse.com/1154661",
					"description": "SUSE Bug 1154661"
				},
				{
					"url": "https://bugzilla.suse.com/1190368",
					"description": "SUSE Bug 1190368"
				},
				{
					"url": "https://bugzilla.suse.com/1191838",
					"description": "SUSE Bug 1191838"
				}
			]
		}
	]
}
